File: Problemset/Problem-B/B2167_YourName.java
import java.util.*;
import java.net.*;
import java.io.*;
import java.math.*;

/**
 * So the problem is check name is same or not
 * 
 * - Using sorting
 * - Compare string first and second
 * - If true return YES and not return FALSE
 */
public class B2167_YourName {
  private static Scanner scanner = new Scanner(System.in);

	public static void main(String[] args) {
    try {
      int testCase = scanner.nextInt();
      while(testCase-- > 0) {
        // write here...
        int length = scanner.nextInt();
        String first = scanner.next();
        String second = scanner.next();

        char[] charArrayFirst = first.toCharArray();
        char[] charArraySecond = second.toCharArray();

        Arrays.sort(charArrayFirst);
        Arrays.sort(charArraySecond);

        String firstSorted = String.valueOf(charArrayFirst);
        String secondSorted = String.valueOf(charArraySecond);
        
        if(firstSorted.equalsIgnoreCase(secondSorted)) {
          System.out.println("YES");
        }
        else {
          System.out.println("NO");
        }
      }
    }
    catch (Exception e) {
      scanner.close();
      System.err.println(e.getMessage());
    }
	}

	static void println(Object o) { System.out.println(o); }

	static void printNumArr2D(int[] n) {
		for(var t : n) {
			System.out.print(t+" ");
		}
		System.out.println();
	}
}

File: Problemset/Problem-B/B2167_your_name.py
import sys

def main():
  try:
    testCase = int(sys.stdin.readline().strip())
    
    for _ in range(testCase):
      line = sys.stdin.readline().strip()
      if not line:
        line = sys.stdin.readline().strip()
      
      first, second = line.split()
      
      if sorted(first.lower()) == sorted(second.lower()):
        print("YES")
      else:
        print("NO")
              
  except Exception as e:
    print(f"Error: {e}", file=sys.stderr)


def println(o):
  print(o)


def printNumArr2D(n):
  for t in n:
    print(t, end=" ")
  print()


if __name__ == "__main__":
  main()